tests/common/utils/test_progress_tracking.py

```diff
@@ -83,4 +83,4 @@ def test_track_context_manager(n, is_weighted):
     with track(total=n, description="Progress...", weights=weights if is_weighted else None) as pbar:
         for i in range(n):
             assert pbar.progress._tasks[pbar.task].completed == (sum(weights[:i]) if is_weighted else i)
-            pbar.progress.update(pbar.task, advance=1)
+            pbar.update(advance=1)
```

Comment:
Why in this case `[None]` is not required for `with track(...)`?

Reply:
it's test, [none] or type ignore need to pass mypy

Reply:
ok